index.html

```diff
@@ -15,31 +15,294 @@
 </head>
 
 <body>
-    <header class="navbar container fixed-top">
-        <a href="" id="logo">
+    <div class="container">
+        <header class="navbar container fixed-top">
+            <a href="" id="logo">
             Welcome
         </a>
-        <nav class="menu">
-            <i class="fa-solid fa-bars "></i>
-        </nav>
-    </header>
-    <section class="header">
-        <div class="container">
-            <h1 class="identity">
-                Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
-            </h1>
-            <p class="profile">
-                <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
-
-            <ul class="icon-group">
-                <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
-                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
-            </ul>
+            <nav class="menu">
+                <i class="fa-solid fa-bars "></i>
+            </nav>
+        </header>
+
+        <section class="header">
+            <div class="container">
+                <h1 class="identity">
+                    Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
+                </h1>
+                <p class="profile">
+                    <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
+
+                <ul class="icon-group">
+                    <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                    <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+                <ul class="icon-group">
+                    <li><i class="fa-solid fa-chevron-down">
+                    </i>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
```

Comment:
- [ ] As required, please kindly use a display of grid on your `works` section container.

Reply:
@topeogunleye 
May you give some clarificatipns plz! I can't get well your suggestions.
Please, respond me at your earliest convience as i don't much time to re-submit my project

Reply:
> @topeogunleye May you give some clarificatipns plz! I can't get well your suggestions. Please, respond me at your earliest convience as i don't much time to re-submit my project. Kindly check [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) for an example:

In your css file, you can add the following:


.works {
  display: grid;
}